nvd3/templates/content.html

```diff
@@ -96,6 +96,12 @@
 
     {% endblock custoattr %}
 
+    {% block y_axis_scale %}
+    {% if chart.y_axis_scale_min or chart.y_axis_scale_max %}
```

Comment:
so if `y_axis_scale_min` is -1 and `chart.y_axis_scale_max` is empty, the next line will produce:

    chart.forceY([-1,]);

Are you sure forceY method will be happy with `[-1,]` (that's the same in JS as `[-1]`) ?